jetbrains-gateway/main.tf

```diff
@@ -80,36 +80,36 @@ variable "jetbrains_ide_versions" {
   description = "The set of versions for each jetbrains IDE"
   default = {
     "IU" = {
-      build_number = "241.14494.240"
```

Comment:
Do we document these changes in any changelog?

Reply:
For now, we write them manually as part of release notes.

https://github.com/coder/modules/releases